specs/trust-store-trust-policy.md

```diff
@@ -415,22 +413,29 @@ Notary Project allows user to execute custom validations during verification usi
 1. **Validate Expiry:**
     1. If an `expiry time` signed attribute is present in the signature envelope, check if the local machine’s current time(in UTC) is greater than `expiry time`. If yes, fail this step.
 1. **Validate Trusted Timestamp:**
```

Comment:
When should we perform timestamp countersignature verification? Should we verify it all the time or only when the signing leaf certificate has expired? Do we prefer one SignerInfo to be valid, or should all SignerInfos be valid? What are the limitations for the timestamp token? Do we need the timestamping certificates (leaf & intermediate) to be included in the token?

Reply:
> When should we perform timestamp countersignature verification? Should we verify it all the time or only when the signing leaf certificate has expired? 

The answer is actually stated two lines below: Under signing scheme `notary.x509`, the `local machine's current time (in  UTC)` is validated against the signing certificate and certificate chain's validity period. If fail, continue to perform the `timestamp countersignature verification`. 

> Do we prefer one SignerInfo to be valid, or should all SignerInfos be valid? What are the limitations for the timestamp token? 

This is `Notary Project` spec, questions above are actually TSA client specific. Answers may be different for different Notary Project implementations. Because each implementation could choose their own tsp clients.

> Do we need the timestamping certificates (leaf & intermediate) to be included in the token?

Yes, as stated in a few lines below, certificates are validated based on Notary Project certificate requirements.

Reply:
Thanks for your explanation. We can resolve the comment.